energy-integration/mex-governance/src/lib.rs

```diff
@@ -0,0 +1,202 @@
+#![no_std]
```

Comment:
If you apply all my "own file" suggestions, maybe have them grouped in a folder called "farm_actions" or something. Keeps the code cleaner, as there are already a lot of files.

Reply:
The entire contract is based on farms, I don't see any real benefits if we add a new folder with a few files (or all) in it. If indeed the SC would have interacted with multiple contract types (like farms, farm-staking, farm-staking-proxy), then yes, it would have structure the code much better.